torch/csrc/jit/test_jit.cpp

```diff
@@ -864,6 +864,23 @@ void testBlocks(std::ostream & out) {
   out << *g2 << "\n";
 }
 
+static std::vector<IValue> run(std::shared_ptr<Graph> graph, std::vector<IValue> stack) {
```

Comment:
Directly running the interpreter bypasses all optimizations and may not always work as you would expect. It also does not error checking and would lead to inscrutable errors. By using jit::script::Module's already existing overloaded operators, you will get the right behavior.